fireapp/lib/presentation/volunteer_information/volunteer_information_widget.dart

```diff
@@ -0,0 +1,50 @@
+import 'dart:convert';
+
+import 'package:fireapp/base/widget.dart';
+import 'package:fireapp/domain/models/volunteer_information.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+import '../../domain/models/reference/qualification.dart';
+
+class volunteer_information_widget extends StatelessWidget {
```

Comment:
This should be named with capitalized camel case -> "VolunteerInformationWidget"